### DISCUSSION TITLE
Timed configuration

### DISCUSSION THREAD

**Discussion by buentAir** [rtl-airband/RTLSDR-Airband#331] [Ideas]
Which would be really cool if the different configurations could be changed over time. For example, there are frequencies that only operate during the day and others at night. Currently I'm solving this manually with the disable=true option. Or does anyone have a great idea with the existing programming?

**Comment by feelingwalnut** [comment]
Run program with -c [desired config file location] argument (check config wiki [here](https://github.com/szpajder/RTLSDR-Airband/wiki/Configuration-essentials)) and specify two cron jobs to run for the desired time periods pointing at the desired config file for that period.

Sometimes you have to, in my experience, write a basic one line script and have the cron run that for a program to run correctly, not sure if airband is one of those.
if needed, that'd be something like an .sh file containing a shebang and the rtlsdr-airband command with the config argument.

The end result would be, cron runs airband with config A from (desired time) EX. 0000-1159 then close and run airband with config B from (desired time) EX. 1200-2359

this should be a "all built in" distro agnostic solution that requires no additional programs.

**Reply by buentAir** [reply]
Thank you for your idea. To make matters worse, I have a VPN connection that interrupts something every hour (FritzBox). Therefore it is difficult to change the configuration file using cron. I like your idea though, maybe I can find a way with it. Thanks very much!
